exercises/none_test.rb

```diff
@@ -15,36 +15,32 @@ def test_none_are_broken
   def test_double_negative
     numbers = [9, 3, 3, 7, 6, -5, 1]
     not_none_negative = numbers.none? do |number|
-      # Your code goes here
+      number + number.abs == 0
     end
     refute not_none_negative
   end
 
   def test_none_are_negative
-    skip
     numbers = [9, 3, 1, 8, 3, 3, 5]
-    # Your code goes here
+    none_negative = numbers.none? { |num| num < 0 }
     assert none_negative
   end
 
   def test_none_shall_pass
-    skip
     critters = ["elf", "hobbit", "dwarf", "wizard", "human"]
-    # Your code goes here
+    none_shall_pass = critters.none? { |w| w == 'pass' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.